src/Raven.Studio/typescript/models/database/tasks/ongoingTaskReplicationHubEditModel.ts

```diff
@@ -99,55 +98,40 @@ class pullReplicationDefinition {
             min: 0
         });
         
-        if (requiresCertificates) {
-            this.certificates.extend({
-                validation: [
-                    {
-                        validator: (cert: Array<pullReplicationCertificate>) => cert.length > 0,
-                        message: "Please define at least one certificate"
-                    }
-                ]
-            });
-
-            this.certificateExported.extend({
-                validation: [
-                    {
-                        validator: (v: boolean) => {
-                            const required = this.certificateGenerated();
-                            return !required || v;
-                        },
-                        message: "Please download or export generated certificate"
```

Comment:
where did you move this logic? Isn't it still needed?

Reply:
this logic is inside `initValidation` inside :  `replicationAccessHubModel.ts`